app/views/organizations/pets/_form.html.erb

```diff
@@ -98,6 +98,14 @@
         <% end %>
       </div>
 
+      <div class='form-group'>
+        <div class="form-check form-switch">
+            <%= form.hidden_field :application_paused, value: false %>
+            <%= form.check_box :application_paused, { class: "form-check-input", role: "switch", id: "flexSwitchCheckChecked" }, true, false %>
+            <%= form.label :application_paused, "Pause application", class: "form-check-label", for: "flexSwitchCheckChecked" %>
+        </div>
+      </div>
```

Comment:
@marlena-b could you update your branch with `main` we recently updated to include bootstrap_form which might make this look strange. Could you double check?

Thanks!

Reply:
Fixed now, thank you!